### DISCUSSION TITLE
Amozesh

### DISCUSSION THREAD

**Discussion by sarvinpakzad** [Ludeon/RimWorld-Farsi#4] [General]
salam mishe amozeshe kameli bezarin /??? daghighan chetori bayad to Rimworld activesh konim ??? mishe to youtube amozeshesh ro ham bezarid ???

**Comment by code-ramtin** [comment]
متشکرم بابت فارسی ساز فقط اگر میشه یک ویدیو اموزشی مربوط درست کردن متنش بزار فارسیش برعکسه

**Reply by DanialPahlavan** [reply]
Hello @code-ramtin,

To address the issue you’ve encountered, please download and use the files from the Release section of our repository. The problem has been resolved in the latest release.

Ensure you’re using version 0.1.6, which contains the necessary fixes. You can find it here: https://github.com/Ludeon/RimWorld-Farsi/releases 
[persian.language.zip](https://github.com/Ludeon/RimWorld-Farsi/releases/download/v0.1.6/persian.language.zip)

**Reply by DanialPahlavan** [reply]
https://www.youtube.com/watch?v=sgoDRB_fIAI
added to readme Farsi
